supertokens_python/recipe/thirdpartyemailpassword/asyncio/__init__.py

```diff
@@ -168,3 +182,51 @@ async def send_email(
     return await ThirdPartyEmailPasswordRecipe.get_instance().email_delivery.ingredient_interface_impl.send_email(
         input_, user_context
     )
+
+
+async def create_reset_password_link(
+    user_id: str,
+    tenant_id: Optional[str] = None,
+    user_context: Optional[Dict[str, Any]] = None,
+):
+    token = await create_reset_password_token(user_id, tenant_id, user_context)
+    if isinstance(token, CreateResetPasswordWrongUserIdError):
+        return CreateResetPasswordLinkUknownUserIdError()
+
+    recipe_instance = ThirdPartyEmailPasswordRecipe.get_instance()
+
+    user = await get_user_by_id(user_id, user_context)
+    assert user is not None
+
+    return CreateResetPasswordLinkOkResult(
+        link=get_password_reset_link(
+            recipe_instance.get_app_info(),
+            token.token,
+            recipe_instance.get_recipe_id(),
+            tenant_id or DEFAULT_TENANT_ID,
+        )
+    )
+
+
+async def send_reset_password_email(
+    user_id: str,
+    tenant_id: Optional[str] = None,
+    user_context: Optional[Dict[str, Any]] = None,
+):
+    link = await create_reset_password_link(user_id, tenant_id, user_context)
+    if isinstance(link, CreateResetPasswordLinkUknownUserIdError):
+        return CreateResetPasswordEmailUnknownUserIdError()
```

Comment:
Should be SendResetPasswordEmail... right?

Reply:
Done